WordPressKit.podspec

```diff
@@ -1,6 +1,6 @@
 Pod::Spec.new do |s|
   s.name          = "WordPressKit"
-  s.version       = "3.1.0"
+  s.version       = "3.1.2"
```

Comment:
I think this would need to be `3.2.0-beta.2` assuming #124 lands first. 

🤔 but now that I think about it, you can @danielebogo could coordinate to roll all the changes into `3.2.0-beta.1` if you were so inclined.

Reply:
@aerych I already merged my PR but I didn't create a tag yet. @jklausa let me know if it's ok for you use `3.2.0-beta.1` with also your PR merged

Reply:
I'll update mine to point to 3.2.0-beta.1 too :)